invenio_previewer/templates/invenio_previewer/macros.html

```diff
@@ -54,7 +54,7 @@
         <td class="nowrap">{{ file.size }}</td>
         <td class="nowrap">
           <span class="pull-right">
-            <a class="btn btn-xs btn-default" href="{{ file.uri }}">
+            <a class="btn btn-xs btn-default" href="{{ url_for('invenio_records_ui.recid_download', pid_value=file.pid.pid_value, filename=file.file.key) }}">
```

Comment:
@lnielsen The links for the file download are a bit hardcoded right now, but the endpoint used in each installation could be different. Previously we were using ``files.uri`` but it is not possible now. How can I calculate it?